src/components/profile-link/profile-link.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import {AppRoute} from '../../const';
+import {Link} from 'react-router-dom';
+import {UserData} from '../../types/user';
+import {selectUserData} from '../../store/user-process/selectors';
+import {useAppSelector} from '../../hooks';
+import {selectFavoritesData} from '../../store/favorites-data/selectors';
+
+function ProfileLink() {
+  const user: UserData = useAppSelector(selectUserData);
+  const favorites = useAppSelector(selectFavoritesData);
+
+  return (
+    <li className='header__nav-item user'>
+      <Link className="header__nav-link header__nav-link--profile" to={AppRoute.favorites}>
+        <div className="header__avatar-wrapper user__avatar-wrapper">
+          <img src={user.avatarUrl} alt={user.name} />
```

Comment:
```suggestion
          <img src={user.avatarUrl} alt="" />
          <VisuallyHidden></VisuallyHidden>
```

https://www.a11yproject.com/posts/how-to-hide-content/

Либо использовать аттрибут `aria-label`
https://doka.guide/a11y/aria-label/

Либо оставить пустым